packages/shared/src/contexts/PaymentContext.tsx

```diff
@@ -89,7 +89,7 @@ export const PaymentContextProvider = ({
             logRef.current({
               event_name: LogEvent.CompleteCheckout,
               extra: {
-                cycle: event?.data.items?.[0]?.billing_cycle.interval,
+                cycle: event?.data.items?.[0]?.billing_cycle?.interval,
```

Comment:
how is this optional now?

Reply:
Interval doesn't exist for one-time payment, so this line was triggering a client error on success

Reply:
🤝